### ISSUE TITLE
doc differences in examples >= 3.20 

### ISSUE THREAD

**Issue by chen23** [F5Networks/f5-appsvcs-extension#283]
### Is your feature request related to a problem? Please describe.
In version 3.20 the examples are updated in a way that no longer work with older versions.  Please add a warning that the examples are specific to version 3.20 and newer and to reference to 3.19 for older examples. 

https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/userguide/examples.html#example-1-simple-http-application

### Describe the solution you'd like
Additional warnings on doc pages.

**Comment by dstokesf5** [comment]
@chen23 We already have warnings all over the docs for this:
* https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/
* https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/http-services.html
* https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/tls-encryption.html
* https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/userguide/faq.html#servmain-ref

**Comment by chen23** [comment]
IMHO the warning should also be on the User Guide examples page in addition to the pages that you've referenced.  Thank you.

**Comment by dstokesf5** [comment]
Good catch, we missed that page. We will get the warning added there as well.
